astropy/utils/typing.py

```diff
@@ -0,0 +1,16 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+Experimental typing support for ``astropy``, subject to change
+without notice.
+"""
+
+from fractions import Fraction
+from typing import TypeAlias
+
+import numpy as np
+
+# The classes from the standard library `numbers` module are not suitable for
+# type checking (https://github.com/python/mypy/issues/3186).
```

Comment:
Just so I can come back in the future and look at the issue status without clicking:

* https://github.com/python/mypy/issues/3186

Reply:
Thanks, that link was helpful. Maybe add
```
# TODO: remove these when a standard way to deal with this has developed
```